### PR TITLE
Management command settings_initialize.

### PR DESCRIPTION
**Pull request by akolpakov**
./manage.py settings_initialize - import DJANGO_SETTINGS into database
Also before this commit post_syncdb handler was not work.

### CODE REVIEW

**Comment by ygneo** [comment]
+1

**Comment by ygneo** [comment]
@jqb It will be nice if you merge this PR. It's passing test suite and the bug is breaking syncdb when using django_settings.

**Comment by akolpakov** [comment]
seems that project is no longer supported.
Now I am using https://github.com/mishbahr/django-usersettings2 instead.

**Comment by jqb** [comment]
Who says it's not supported?! :)

File: django_settings/dataapi.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # module
 from .cache import cache_method, MethodProxy
 from .lazyimport import lazyimport
+import conf
```
Comment (jqb) [inline django_settings/dataapi.py]: please import relatively as it is with "lazyimport"

File: django_settings/models.py [hunk 2]
```diff
@@ -4,6 +4,9 @@
 from django.contrib.contenttypes.models import ContentType
 from django.contrib.contenttypes import generic
 from django.utils.translation import ugettext_lazy as _
+from django.dispatch import receiver
+from django.db.models.signals import post_syncdb
+from dataapi import initialize_data
```
Comment (jqb) [inline django_settings/models.py]: relative import (as the one below for "from . import conf") will be better here as welll

**Comment by ygneo** [comment]
@jqb you can prove it is actually supported! I mean, it will be nice if you merge this and update the package in PyPi. :-)

**Comment by jqb** [comment]
Sure. Can anybody make those small changes I've just commented on?

File: django_settings/models.py (hunk 2 above)
Comment (ygneo) [inline django_settings/models.py]: You mean for line 9, don't you? Lines 7 & 8 are absolute imports of django packages.

**Comment by jqb** [comment]
Ok, it's not that much I'll do it just after merge.

**Comment by ygneo** [comment]
Nice! I was going to do it, thanks. Could you ping when a new release is in PyPi?

**Comment by jqb** [comment]
https://pypi.python.org/pypi/django-settings/1.3.12

**Comment by ygneo** [comment]
Thanks a lot! :-)

**Pull request merged**